cmd/controller-manager/app/controller-manager.go

```diff
@@ -100,6 +100,17 @@ func Run(opts *options.Options) error {
 
 	stopChan := setupSignalHandler()
 
+	// TODO(sohankunkerkar) Remove when no longer necessary.
+	// This Environment variable is a temporary addition to support Red Hat's downstream testing efforts.
+	// Its continued existence should not be relied upon.
+	defaultValue := os.Getenv("DEFAULT-SCOPE")
```

Comment:
Please name the env var `DEFAULT_FEDERATION_SCOPE`.  Dashes (`-`) should never be used in the name of an env var - only ascii characters, numbers and underscores (`_`).